### DISCUSSION TITLE
I wish there is a support for multiple accounts in mobile github app.

### DISCUSSION THREAD

**Discussion by A-BCDe** [community/community#19989] [Enterprise] [answered]
Although there is a partial support for enterprise accounts, it would be nice if the app supports logging in with multiple users.

**Comment by elisealix22** [comment]
Hi @A-BCDe , thank you so much for the request! This is a highly requested feature and something that we are absolutely tracking internally. Out of curiosity, are you trying to sign in to two personal accounts or is one for work and one personal?

**Reply by A-BCDe** [reply]
I'd like to sign in with multiple accounts.

**Reply by dotku** [reply]
My situation is my company is using GitHub instead of Enterprice version, so it would be good to have two GitHub accounts switchable.

**Reply by smil2k** [reply]
> Hi @A-BCDe , thank you so much for the request! This is a highly requested feature and something that we are absolutely tracking internally. Out of curiosity, are you trying to sign in to two personal accounts or is one for work and one personal?

I have one enterprise and one personal account. But the enterprise account is on the cloud hosted variant counts as a personal account...

**Reply by mattwiley-8451** [reply]
I have multiple free-tier personal accounts (yes, I need to consolidate... all in due time) and a work account in GHEC (current account).

Immediate ask: I would love to be able to switch between them in the app.

Nice to have: Would be even better if they could all be configured simultaneously so I could receive notifications from each of them.

**Comment by ToastShaman** [comment]
> Although there is a partial support for enterprise accounts, it would be nice if the app supports logging in with multiple users.

I wish this would be possible as well. I would use it to login with 2 "personal" accounts. One I use for my side projects, weekend experiments and one account I use for work which is part of client organisations.

**Comment by eliperkins** [comment, marked as answer]
Hey @A-BCDe and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Reply by jsoref** [reply]
@eliperkins can you mark this as the answer so it stops showing up as an unanswered item?
